spec/fixtures/state_file/fed_return_jsons/2023/id/tim_1099_int.json

```diff
@@ -0,0 +1,26 @@
+{
+  "familyAndHousehold": [],
+  "filers": [
+    {
+      "firstName": "Tim",
+      "middleInitial": "T",
+      "lastName": "Interest",
+      "dateOfBirth": "1980-01-01",
+      "isPrimaryFiler": true
```

Comment:
can the filer also have a `tin` field as well since that's something we should always expect (hopefully) from IRS?

Reply:
thanks i'll add in the tin values!